packages/cli/src/constructs/alert-escalation-policy.ts

```diff
@@ -1,33 +1,113 @@
+/** Types of alert escalation strategies */
 // eslint-disable-next-line no-restricted-syntax
 enum AlertEscalationType {
   RUN = 'RUN_BASED',
   TIME = 'TIME_BASED'
 }
 
+/**
+ * Configuration for alert reminders.
+ * Defines how often to send reminder notifications after initial alert.
+ */
 export type Reminders = {
+  /** 
+   * Number of reminder notifications to send (0 to disable).
+   * @defaultValue 0
+   * @minimum 0
+   * @maximum 5
+   * @example 2  // Send 2 reminder notifications
+   */
   amount?: number,
+  
+  /** 
+   * Interval between reminder notifications in minutes.
+   * @defaultValue 5
+   * @enum [5, 10, 15, 30]
+   */
   interval?: number
 }
 
+/**
+ * Configuration for parallel run failure threshold.
+ * Determines when to alert based on percentage of failed parallel runs.
+ */
 export type ParallelRunFailureThreshold = {
+  /** 
+   * Whether parallel run failure threshold is enabled.
+   * @defaultValue false
+   */
   enabled?: boolean,
+  
+  /** 
+   * Percentage of runs that must fail to trigger alert.
+   * @defaultValue 10
+   * @minimum 10
+   * @maximum 100
+   * @multipleOf 10
+   * @example 30  // Alert when 30% of parallel runs fail
+   */
   percentage?: number,
 }
 
+/**
+ * Configuration for alert escalation policies.
+ * Defines when and how to escalate alerts based on check failures.
+ */
 export interface AlertEscalation {
+  /** The type of escalation strategy to use */
   escalationType?: AlertEscalationType,
+  /** Configuration for run-based escalation */
   runBasedEscalation?: {
+    /** 
+     * Number of consecutive failed runs before escalating.
+     * @defaultValue 1
+     * @minimum 1
+     * @maximum 5
+     * @example 3  // Escalate after 3 consecutive failures
+     */
     failedRunThreshold?: number
   },
+  /** Configuration for time-based escalation */
   timeBasedEscalation?: {
+    /** 
+     * Minutes that check must be failing before escalating.
+     * @defaultValue 5
```

Comment:
Similar to above, the backend doesn't seem to enforce a default value. If `AlertEscalationBuilder` is used, then the default is `5`.

Reply:
I think it's reasonable to keep the default value, and we can try to fix it in the BE to apply even if the user does not use the builder. That's more consistent anyway.